pkg/mqtt5/bootstrap/container.go

```diff
@@ -38,34 +39,34 @@ func (mc *Mqtt5ClientMap) Get(brokerName string) (mqtt5.Mqtt5Client, error) {
 
 	for name, client := range mc.mqtt5Clients {
 		if name == brokerName {
-			return client, nil
+			return *client, nil
```

Comment:
why not using the map to retrieve the client instead of for loop?

Reply:
updated